src/marketing/products-viewed-compared.md

```diff
@@ -12,13 +12,19 @@ The Recently Viewed and Recently Compared blocks usually appear in the right sid
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Recently Viewed/Compared Products** section and do the following:
 
-    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-compared-products.png %}){: .zoom}
+    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-and-compared-products.png %}){: .zoom}
     [_Recently Viewed/Compared Products_]({% link configuration/catalog/catalog.md %})
 
+   - Set **Synchronize widget products with backend storage** to synchronize product widget information such as product id with your current product storage availability in database. In order to reuse this info on different devices.
+
    - Set **Show for Current** to the website, store, or store view where the configuration applies.
 
    - In the **Default Recently Viewed Products Count** field, enter the number of recently viewed products to appear in the list.
 
    - In the **Default Recently Compared Products Count**, enter the number of recently compared products to appear in the list.
 
+   - In the **Lifetime of products in Recently Viewed Widget** enter how long the products viewed will be shown in the recently viewed list.
```

Comment:
The default value for these fields is `1000`... what does that equal? 1000 days, hours, something else?

Reply:
Hi @leslietilling, thank you for the great question!

According to the code, the default value for widgets is 1000 seconds
**Source code:**
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/Model/FrontendStorageConfigurationInterface.php#L20
**Regarding the range:**
There is no limit on the maximum amount of time, but at the same time, there is validation - only numbers and no less than zero.

`],
        ‘validate-greater-than-zero’: [
            function (v) {
                if ($.mage.isEmptyNoTrim(v)) {
                    return true;
                }
                v = $.mage.parseNumber(v);
                return !isNaN(v) && v > 0;
 },
            $.mage.__(‘Please enter a number greater than 0 in this field.’)
        ]`
**Source code:**
https://github.com/magento/magento2/blob/2.4/lib/web/mage/validation.js#L823

For more transparency for customers, I'll add clarification to the files and let you know. :)

Reply:
@leslietilling,
I have implemented some edits for src/marketing/products-viewed-compared.md  file. Could you please review and approve?

Thanks!